components/infobox/commons/infobox_character.lua

```diff
@@ -0,0 +1,123 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:Infobox/Character
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Namespace = require('Module:Namespace')
+local BasicInfobox = require('Module:Infobox/Basic')
+local Flags = require('Module:Flags')
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+local Header = Widgets.Header
+local Title = Widgets.Title
+local Center = Widgets.Center
+local Customizable = Widgets.Customizable
+
+local Character = Class.new(BasicInfobox)
+
+function Character.run(frame)
+	local character = Character(frame)
+	return character:createInfobox()
+end
+
+function Character:createInfobox()
+	local infobox = self.infobox
+	local args = self.args
+
+	local widgets = {
+		Header{
+			name = self:nameDisplay(args),
+			subHeader = self:subHeader(args),
+			image = args.image,
+			imageDefault = args.default,
+			imageDark = args.imagedark or args.imagedarkmode,
+			imageDefaultDark = args.defaultdark or args.defaultdarkmode,
+			size = args.imagesize,
+		},
+		Center{content = {args.caption}},
+		Title{name = (args.informationType or 'Hero') .. ' Information'},
+		Cell{name = 'Real Name', content = {args.realname}},
+		Customizable{
+			id = 'country',
+			children = {
+				Cell{
+					name = 'Country',
+					content = {
+						self:_createLocation(args.country)
+					}
+				},
+			}
+		},
+		Customizable{
+			id = 'role',
+			children = {
+				Cell{
+					name = 'Role',
+					content = {args.role}
+				},
+			}
+		},
+		Customizable{
+			id = 'class',
+			children = {
+				Cell{
+					name = 'Class',
+					content = {args.class}
+				},
+			}
+		},
+		Customizable{
+			id = 'release',
+			children = {
+				Cell{
+					name = 'Release Date',
+					content = {args.releasedate}
+				},
+			}
+		},
+		Customizable{id = 'custom', children = {}},
+		Center{content = {args.footnotes}},
+	}
+
+	infobox:categories('Characters')
+	infobox:categories(unpack(self:getWikiCategories(args)))
```

Comment:
do we want to always set the catrgories, or only in main space?

Reply:
I'm thinking its best to have it added in all spaces rather than just main.

Reply:
Categories are usually only added to main space